### DISCUSSION TITLE
手抖寶貝看過來 ! 防手抖口紅輔助器

### DISCUSSION THREAD

**Discussion by yuxinlin93** [1101-GS4719-C/Python-Community-GS4719-C#1096] [General]
昨天在新聞上看到這一個輔助器，一開始看看覺得有點瞎，有需要為了畫口紅那麼大費周章弄一個輔助器嗎
但發現這一個輔助器不只可以幫助平常那一些手不太穩的女孩畫口紅，也可以幫助有一些有手部疾病或無法控制動作的人，其實是很有意義
像是之前有一個妥瑞氏症女孩因為在影音平台分享她平常化妝的過程而爆紅，過程中他很常因為一次一次無法控制的抽動畫歪，還要慢慢擦掉歪掉的地方再繼續重畫。他表現得非常樂觀，但辛苦的過程不免讓人看得有些心疼。而這個輔助器的發明可以為有這種需求的人手下許多擦掉重畫的時間，可以說是非常方便。
[https://news.knowing.asia/news/b0db62c2-0de2-464b-b2a9-62108c6efe15](url)
[https://www.youtube.com/watch?v=F88kzxTGNN0](url)

**Comment by Valentinaa0214** [comment]
很有意義的發明欸！有點像相機穩定器的感覺，希望未來能夠有更多新創發明延續這個風氣，
造福那些有先天性肢體障礙或後天疾病導致身體無法像正常人活動的人們，讓他們能夠便利的生活，不被自身殘缺所限制

**Comment by wenchi9023** [comment]
剛剛看到你的標題而點進來的，想說是甚麼無用小廢物，
結果原來另有意義，我太失敬了!
話說原PO的連結好像失效了~
